README.rst

```diff
@@ -11,9 +11,28 @@ Installing
 * Gentoo users: ``emerge app-vim/gentoo-syntax``
 * Everyone else: ``make PREFIX=~/.vim/ install``
 
+Syntax checker
+--------------
```

Comment:
Let's keep it as the same level as 'installing'.

Could you also mention reading syntastic's README for generic setup?

Reply:
Done. Should I add hyperlinks too?

Reply:
Sure, why not.